core/src/main/java/org/elasticsearch/index/query/ConstantScoreQueryBuilder.java

```diff
@@ -29,31 +32,31 @@
  * A query that wraps a filter and simply returns a constant score equal to the
  * query boost for every document in the filter.
  */
-public class ConstantScoreQueryBuilder extends QueryBuilder implements BoostableQueryBuilder<ConstantScoreQueryBuilder> {
+public class ConstantScoreQueryBuilder extends QueryBuilder<ConstantScoreQueryBuilder> implements BoostableQueryBuilder<ConstantScoreQueryBuilder> {
 
     public static final String NAME = "constant_score";
 
     private final QueryBuilder filterBuilder;
 
-    private float boost = -1;
+    private float boost = 1.0f;
```

Comment:
I noticed that this query doesn't support the queryName also... given that I am working on that, we can skip adding it, it will get it for free when mvoed to the base class. Ok?